### PR TITLE
fix Dockerfile to create alpine based docker image

### PR DESCRIPTION
**Pull request by hayajo**
- change baseimage to alpine:3.7

### CODE REVIEW

**Comment by shibayu36** [comment]
We should fix PR title to understand what this PR do by fixing Dockerfile.

File: Dockerfile [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
-ENTRYPOINT ["/go/bin/app"]
+FROM golang:alpine3.7
+RUN apk add --no-cache make git
+WORKDIR /go/src/github.com/mackerelio/mkr/
+COPY . .
+RUN make build
+
+FROM alpine:3.7
+RUN apk add --no-cache ca-certificates
+WORKDIR /
```
[the remark this answers is not in the document]
Comment (hayajo) [reply Dockerfile]: `WORKDIR /` isn't needed. I'll remove it.

File: Dockerfile [hunk 2]
```diff
@@ -1,2 +1,11 @@
...
+COPY --from=0 /go/src/github.com/mackerelio/mkr/mkr /usr/local/bin/
+ENTRYPOINT ["mkr"]
```
[the remark this answers is not in the document]
Comment (hayajo) [reply Dockerfile]: I think so, too. fix it.

**Comment by shibayu36** [comment]
https://github.com/mackerelio/mkr/blob/fix-dockerfile/README.md#linux-docker is old.  Fix it, please.

**Comment by shibayu36** [comment]
LGTM!!

**Comment by hayajo** [comment]
Tank you!

**Pull request merged**